src/screens/tabs/home/Home.js

```diff
@@ -2,28 +2,20 @@ import React from "react";
 import { Text, View, StyleSheet, Button } from "react-native";
 import ItemList from "../../commons/itemList/ItemList";
 import { HomeHeader } from "../../../components/Headers";
+import ModalTest from "../../../components/modalTest";
 
 const Home = ({ navigation }) => {
+  const goSetMyTown = () => navigation.push("SetMyTown");
+  const goInterestedCategory = () => navigation.push("InterestedCategory");
+  const goHomeSearch = () => navigation.push("HomeSearch");
+  const goItemDetail = () => navigation.push("ItemDetail");
   return (
     <>
-      <HomeHeader></HomeHeader>
-      <View style={styles.container}>
-        <Button
-          title="go to SetMyTown"
-          onPress={() => {
-            navigation.push("SetMyTown");
-          }}
-        />
-        <Button
-          title="go to InterestedCategory"
-          onPress={() => navigation.push("InterestedCategory")}
-        />
-        <Button
-          title="go to HomeSearch"
-          onPress={() => navigation.push("HomeSearch")}
-        />
-      </View>
-      <ItemList route={{ navigation }} />
+      <HomeHeader
+        props={{ goSetMyTown, goHomeSearch, goInterestedCategory }}
```

Comment:
props 넘기실 때 props 말고 각각 넘겨 주세요!
```suggestion
        goSetMyTown={goSetMyTown}
        goHomeSearch={goHomeSearch}
        goInterestedCategory={goInterestedCategory}
```

Reply:
`goScreens={{ goSetMyTown, goHomeSearch, goInterestedCategory }}`
하는건 좋은 방법이 아닌가요?
함수들을 하나로 묶어서 넘겨주고 싶었습니다.

Reply:
오 그럼 goScreens라는 이름으로 하시면 좋을 것 같네요! props이름이 props인 건 개인적으로는 의미가 불명확해서 goScreens으로 하면 괜찮을 것 같네요!